chapters/ch03.md

```diff
@@ -1,26 +1,30 @@
 # Prerequisites and Notation
 
 ->
+# Điều kiện tiên quyết và Chú ý
 
 
 If you have taken a Machine Learning course such as my machine learning MOOC on Coursera, or if you have experience applying supervised learning, you will be able to understand this text.
 
--> 
+->
+Nếu bạn đã từng học một lớp về Machine Learning ví dụ như lớp MOOC của tôi trên Coursera, hoặc là bạn đã từng có kinh nghiệm trong việc áp dụng *học có giám sát* thì nội dung của cuốn sổ tay này sẽ là dễ hiểu đối với bạn.
 
 
 I assume you are familiar with ​**supervised learning​**: learning a function that maps from x to y, using labeled training examples (x,y). Supervised learning algorithms include linear regression, logistic regression, and neural networks. There are many forms of machine learning, but the majority of Machine Learning’s practical value today comes from supervised learning.
 
 ->
-
+Tôi giả định rằng bạn đã quen thuộc với *học có giám sát*: học một phương trình được gán từ x tới y, sử dụng các ví dụ về nhãn được định nghĩa từ trước (x,y). Các thuật toán của *học có giám sát* bao gồm *hồi quy tuyến tính, logistic regression và mạng neural.* Machine learning có rất nhiều biến thể tuy nhiên phần lớn các giá trị thực tiễn của nó hiện nay đến từ *học có giám sát*.
```

Comment:
- "phương trình ~~được~~ gán từ x tới y"
-  "sử dụng các ví dụ về nhãn được định nghĩa từ trước (x,y)" -> "sử dụng các ví dụ huấn luyện có đính kèm nhãn (x,y)"
- "Các thuật toán ~~của~~ *học có giám sát*"
- "hồi quy tuyến tính, logistic regression" -> mình nghĩ nên giữ nguyên logistic regression hoặc chuyển thành "hồi quy tuyến tính, hồi quy logistic"

Reply:
* Từ "map" trong toán hay được dịch là "ánh xạ"
* "using labeled training examples" -> "sử dụng  các cặp dữ liệu có nhãn". Dịch thế này xuôi tai hơn.
* Những từ mà bản gốc không in nghiêng hay in đậm thì bản dịch cũng nên giữ nguyên như thế. Nếu bản gốc được copy sai vào đây, bạn có thể sửa cả bản gốc.
* Mình đồng ý nên chuyển "linear regression, logistic regression" thành "hồi quy tuyến tính, hòi quy logistic".
* Từ "forms" không nên dịch là "biến thể" vì như thế dễ nhầm với "variants". Mình nghĩ dịch là "dạng" hợp lý hơn.